features/support/github.js

```diff
@@ -0,0 +1,198 @@
+ * @param  {Number}     prNumber           Number of the pull request
+ * @return {Promise}
+ */
+function closePullRequest(token, repoOwner, repoName, prNumber) {
+    github.authenticate({
+        type: 'oauth',
+        token
+    });
+
+    return github.pullRequests.update({
+        user: repoOwner,
+        repo: repoName,
+        number: prNumber,
+        state: 'closed'
+    });
+}
+
+/**
+ * Create a branch on the given repository
+ * @method createBranch
+ * @param  {String}     token              Github token
+ * @param  {String}     branch             Name of the branch to create
+ * @param  {String}     [repoOwner]        Owner of the repository
+ * @param  {String}     [repoName]         Name of the repository
+ * @return {Promise}
+ */
+function createBranch(token, branch, repoOwner, repoName) {
+    const user = repoOwner || 'screwdriver-cd';
+    const repo = repoName || 'garbage-repository-ignore-this';
+
+    // Branch creation requires authentication
+    github.authenticate({
+        type: 'oauth',
+        token
+    });
+
+    // Create a branch from the tip of the master branch
+    return github.gitdata.getReference({
+        user,
+        repo,
+        ref: 'heads/master'
+    })
+    .then((referenceData) => {
+        const sha = referenceData.object.sha;
+
+        return github.gitdata.createReference({
+            user,
+            repo,
+            ref: `refs/heads/${branch}`,
+            sha
+        });
+    });
+}
+
+/**
+ * Creates a random file, with a random content.
+ * @method createFile
+ * @param  {String}   token             Github token
+ * @param  {String}   branch            The branch to create the file in
+ * @param  {String}   repoOwner         Owner of the repository
+ * @param  {String}   repoName          Name of the repository
+ * @return {Promise}
+ */
+function createFile(token, branch, repoOwner, repoName) {
+    const content = new Buffer(randomString(MAX_CONTENT_LENGTH));
+    const filename = randomString(MAX_FILENAME_LENGTH);
+    const repo = repoName;
+    const user = repoOwner;
+
+    github.authenticate({
+        type: 'oauth',
+        token
+    });
+
+    return github.repos.createFile({
+        user,
+        repo,
+        path: filename,
+        message: (new Date()).toString(),    // commit message is the current time
+        content: content.toString('base64'), // content needs to be transmitted in base64
+        branch
+    });
+}
+
+/**
+ * Creates a pull request.
+ * @method createPullRequest
+ * @param  {String}   token             Github token
+ * @param  {String}   branch            The branch to create the file in
+ * @param  {String}   repoOwner         Owner of the repository
+ * @param  {String}   repoName          Name of the repository
+ * @return {Promise}
+ */
+function createPullRequest(token, branch, repoOwner, repoName) {
+    const repo = repoName;
+    const user = repoOwner;
```

Comment:
What do you mean? Can't you just put:
```js
return github.pullRequests.create({
   user: repoOwner, 
   repo: repoName, 
   ...
});
```
Just like this one: https://github.com/screwdriver-cd/screwdriver/blob/575fd0205959a58b7aaeb040eda6ea54e2f72f3d/features/support/github.js#L184-L188